go.mod

```diff
@@ -17,36 +17,34 @@ require (
 	github.com/prometheus/common v0.32.1
 	github.com/spf13/cobra v1.4.0
 	google.golang.org/grpc v1.47.0
-	k8s.io/api v0.25.0
-	k8s.io/apimachinery v0.25.0
-	k8s.io/client-go v0.25.0
+	k8s.io/api v0.25.2
+	k8s.io/apimachinery v0.25.2
+	k8s.io/client-go v0.25.2
 	k8s.io/component-base v0.25.0
 	k8s.io/component-helpers v0.25.0
-	k8s.io/klog/v2 v2.70.1
+	k8s.io/klog/v2 v2.80.1
 	k8s.io/kubernetes v1.25.0
 )
```

Comment:
Line 26 should be 1.25.2
Also line 23 an 24.

Reply:
done!